scripts/o3de/ExportScripts/export_source_ios_xcode.py

```diff
@@ -14,24 +14,24 @@
 from o3de.export_project import process_command
 import pathlib
 
-def export_ios_xcode_project(ctx: exp.O3DEScriptExportContext,
+def export_ios_Xcode_project(ctx: exp.O3DEScriptExportContext,
```

Comment:
The X should remain lowercase to be consistent with [PEP 8](https://pep8.org/#descriptive-naming-styles) which is the convention our python code uses
```suggestion
def export_ios_xcode_project(ctx: exp.O3DEScriptExportContext,
```

Reply:
I must have been sleep deprived to let that sneak in... fixing it now